### DISCUSSION TITLE
Make a demo site

### DISCUSSION THREAD

**Discussion by robin-dongbin** [filamentphp/filament#258] [Ideas]
If there is a demo site, that's will be nice. I have shared this package in Chinese Laravel forum. A demo site can help users know features quickly

**Comment by danharrin** [comment]
Hey! We have a demo app repository - [laravel-filament/demo](https://github.com/laravel-filament/demo).

**Reply by robin-dongbin** [reply]
I know. If you deploy it online, that's will be better

**Comment by blite** [comment]
looks like this repo is dead, I assume because of v2, would be great to have it back again

**Reply by danharrin** [reply]
Yeah, I'd also like a demo of each package on the site as well. It'll probably be a live demo too.
